### ISSUE TITLE
⭐FEAT: Enhance the logo name in header 

### ISSUE THREAD

**Issue by ArslanYM** [ArslanYM/StarterHive#246]
# TODO: 
- The logo name is header looks something like this rn : 
 -  ![s](https://github.com/ArslanYM/StarterHive/assets/104521101/9a6a60a7-15f0-49f6-b9ea-63fc1f3b35c9)

- The goal is to make it better looking , maybe add a new logo , animation.

**Comment by ArslanYM** [comment]
@abhisheksharma886 Sure thing! Assigned to you.

**Comment by shubhjhawar** [comment]
@ArslanYM is this issue still pending? if yes I would like to work on it, if not, can you please assign me some other work. Thanks

**Comment by ArslanYM** [comment]
Sure @shubhjhawar , Do you have anything in mind ?

**Comment by ArslanYM** [comment]
A good feature would be creating an svg for this.

**Comment by ArslanYM** [comment]
> A good feature would be creating an svg for this.

Create a logo for this with better styling or improve styling for the existing one.

**Comment by shubhjhawar** [comment]
sure, can I get started on it?

**Comment by shubhjhawar** [comment]
<img width="439" alt="image" src="https://github.com/ArslanYM/StarterHive/assets/67875612/e340bb67-74d1-4875-97ad-f53d49c39248">
do you like anyone from these?

**Comment by shubhjhawar** [comment]
@ArslanYM shall i use the first one and make a pull request?

**Comment by ArslanYM** [comment]
@shubhjhawar This is not what we are looking for, something minimal and with a logo. 
Since it will be going in the navbar, we want an actual element to be placed there

**Comment by shubhjhawar** [comment]
i can get the svg of any of the above logo and use it in our project, would that work?
or are you looking for something specific??

**Comment by ArslanYM** [comment]
> i can get the svg of any of the above logo and use it in our project, would that work? or are you looking for something specific??

Yeah im looking for something minimal, not how the above are designed

**Comment by shubhjhawar** [comment]
i am sorry I cant design a logo on my own, can I look for some other work in the project?

**Comment by ArslanYM** [comment]
@shubhjhawar Ofcourse, Look for issues you can work on in the issues section.

**Comment by inbuilt-aura** [comment]
@ratishjain12 Hey, is this issue open.If yes, i want to work on this issue.

**Comment by ArslanYM** [comment]
@inbuilt-aura Yes this issue is open
